cypress/integration/HW15/web_tables.spec.js

```diff
@@ -0,0 +1,179 @@
+import { RANDOM_NAME, RANDOM_LAST_NAME, RANDOM_EMAIL, RANDOM_DEPARTMENT, RANDOM_AGE, RANDOM_SALARY, EDIT_RANDOM_AGE, EDIT_RANDOM_DEPARTMENT, EDIT_RANDOM_EMAIL, EDIT_RANDOM_NAME, EDIT_RANDOM_LAST_NAME, EDIT_RANDOM_SALARY } from '../../pages/web_tables/variables.mjs';
+import * as locators from '../../pages/web_tables/locators.mjs';
+import { enteringSearchDataFunction, checkingSearchDataFunction, notSortedArrayFunc, sortedArrayFunc } from '../../pages/web_tables/cypress_functions.mjs';
+
+describe('Add a new user on the "Web table" page', () => {
+    before(() => {
+        cy.visit('/webtables');
+        cy.clearLocalStorage();
+        cy.clearCookies();
```

Comment:
In general, this commands need to use for logout, or for clearing cookies in beforeeach.

Reply:
Done